platform-ui/plugins/org.eclipse.ui.examples.statushandlers.testtool/META-INF/MANIFEST.MF

```diff
@@ -16,6 +16,4 @@ Export-Package: org.eclipse.ui.examples.statushandlers.testtool,
  org.eclipse.ui.examples.statushandlers.testtool.views,
  org.eclipse.ui.examples.statushandlers.testtool.wizards
 Bundle-Localization: plugin
-Bundle-RequiredExecutionEnvironment: J2SE-1.4,
- CDC-1.0/Foundation-1.0,
- J2SE-1.3
+Bundle-RequiredExecutionEnvironment: CDC-1.0/Foundation-1.0
```

Comment:
Why is the BREE for this one different than the rest? Should it be:

```suggestion
Bundle-RequiredExecutionEnvironment: JavaSE-17
```

Reply:
good finding, thanks.